internal/telemetry/client.go

```diff
@@ -195,7 +203,14 @@ func (c *Client) Start(integrations []Integration, configuration []Configuration
 	if c.SubmissionInterval == 0 {
 		c.SubmissionInterval = 60 * time.Second
 	}
-	c.t = time.AfterFunc(c.SubmissionInterval, c.backgroundFlush)
+	c.flushT = time.AfterFunc(c.SubmissionInterval, c.backgroundFlush)
+
+	heartbeat := internal.IntEnv("DD_TELEMETRY_HEARTBEAT_INTERVAL", defaultHeartbeatInterval)
+	if heartbeat < 1 || heartbeat > 3600 {
+		heartbeat = defaultHeartbeatInterval
```

Comment:
Can we log a warning here that the provided value was either too large or too small? It's unlikely anyone will run into this, but a warning will ensure anyone who accidentally messes this up will have an obvious indication of what they did wrong